app/views/insured/interactive_identity_verifications/_insured_ridp_modal.html.erb

```diff
@@ -1,35 +1,94 @@
+                <legend><%= l10n("insured.consumer_roles.upload_ridp_documents.document_type") %></legend>
                 <span>
-                  <label class="mb-4" for="ridp_subject_Driver_s_License_issued_by_state_or_territory"><%= radio_button_tag 'ridp_subject', "Driver's License issued by state or territory", true %><%= l10n("insured.consumer_roles.upload_ridp_documents.driver_license") %></label>
-                  <label class="mb-4" for="ridp_subject_School_identification_card"><%= radio_button_tag 'ridp_subject', 'School identification card', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.school_id") %></label>
-                  <label class="mb-4" for="ridp_subject_U.S._military_card_or_draft_record"><%= radio_button_tag 'ridp_subject', 'U.S. military card or draft record', false%><%= l10n("insured.consumer_roles.upload_ridp_documents.military_id") %></label>
-                  <label class="mb-4" for="ridp_subject_Identification_card_issued_by_the_federal__state_or_local_government__including_a_U.S._Passport"><%= radio_button_tag 'ridp_subject', 'Identification card issued by the federal, state or local government, including a U.S. Passport', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.passport") %></label>
-                  <label class="mb-4" for="ridp_subject_Military_dependent_s_identification_card"><%= radio_button_tag 'ridp_subject', "Military dependent's identification card", false %><%= l10n("insured.consumer_roles.upload_ridp_documents.military_dependent") %></label>
-                  <label class="mb-4" for="ridp_subject_Native_American_tribal_document"><%= radio_button_tag 'ridp_subject', 'Native American tribal document', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.native_american") %></label>
-                  <label class="mb-4" for="ridp_subject_U.S._Coast_Guard_Merchant_Mariner_card"><%= radio_button_tag 'ridp_subject', 'U.S. Coast Guard Merchant Mariner card', false %><%= l10n("insured.consumer_roles.upload_ridp_documents.coast_guard") %></label>
+                  <% options = {
+                    :driver_license => "Driver's License issued by state or territory", 
+                    :school_id => "School identification card", 
+                    :military_id => "U.S. military card or draft record", 
+                    :passport => "Identification card issued by the federal, state or local government, including a U.S. Passport", 
+                    :military_dependent => "Military dependent's identification card", 
+                    :native_american => "Native American tribal document", 
+                    :coast_guard => "U.S. Coast Guard Merchant Mariner card"
+                    } %>
+                  <% element_name = "ridp_subject" %>
+                  <% options.each_with_index do |(option, input_value), index| %>
+                    <div class="d-inline-flex flex-row-reverse align-items-center mb-2">
```

Comment:
this ended up being a little more verbose than intended... the legacy code for the label/radio was much simpler _but_ unlike legacy, the label doesn't wrap under the radio in our mockups (this really is only apparent on the `passport` label in the mockups). Ended up cutting down on repeating this div by using a for loop which helps a little